src/main/java/com/github/hokkaydo/eplbot/module/code/CodeCommand.java

```diff
@@ -0,0 +1,174 @@
+package com.github.hokkaydo.eplbot.module.code;
+import com.github.hokkaydo.eplbot.Config;
+import com.github.hokkaydo.eplbot.Strings;
+import com.github.hokkaydo.eplbot.command.Command;
+import com.github.hokkaydo.eplbot.command.CommandContext;
+
+import net.dv8tion.jda.api.entities.Guild;
+import net.dv8tion.jda.api.entities.channel.middleman.MessageChannel;
+import net.dv8tion.jda.api.events.interaction.ModalInteractionEvent;
+import net.dv8tion.jda.api.hooks.ListenerAdapter;
+import net.dv8tion.jda.api.interactions.InteractionType;
+import net.dv8tion.jda.api.interactions.commands.OptionType;
+import net.dv8tion.jda.api.interactions.commands.build.OptionData;
+import net.dv8tion.jda.api.interactions.components.text.TextInput;
+import net.dv8tion.jda.api.interactions.components.text.TextInputStyle;
+import net.dv8tion.jda.api.interactions.modals.Modal;
+import net.dv8tion.jda.api.interactions.modals.ModalMapping;
+import net.dv8tion.jda.api.utils.FileUpload;
+import org.jetbrains.annotations.NotNull;
+
+import java.util.Optional;
+import java.util.function.Supplier;
+import java.io.FileWriter;
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.stream.Collectors;
+import java.util.List;
+import java.util.Objects;
+import java.time.Instant;
+import java.util.Map;
+import java.util.HashMap;
+
+
+public class CodeCommand extends ListenerAdapter implements Command{
+    private static final Map<String, Runner> RUNNER_MAP = new HashMap<>();
+    private static final String TEMP_DIR = System.getProperty("user.dir")+"\\src\\temp";
+
+    @Override
+    public void executeCommand(CommandContext context) {
+        RUNNER_MAP.put("python", new PythonRunner());
+        RUNNER_MAP.put("rust", new RustCompiler());
+        RUNNER_MAP.put("java", new JavaRunner());
+
+        Guild guild = context.interaction().getGuild();
+        if(guild == null) return;
+
+        if (context.options().size() <= 1) {
+            context.interaction().replyModal(Modal.create(context.author().getId() + "submitCode","Execute du code")
+                                                     .addActionRow(TextInput.create("language", "Language", TextInputStyle.PARAGRAPH).setPlaceholder("python|rust|java").setRequired(true).build())
+                                                     .addActionRow(TextInput.create("body", "Code", TextInputStyle.PARAGRAPH).setPlaceholder("Code").setRequired(true).build())
+                                                     .build()).queue();
+        } else {
```

Comment:
Use `if ... return` instead of `if-else`. It remove an indentation level and give a better readability